elephant/run.sh

```diff
@@ -1,7 +1,25 @@
 #!/bin/bash
 
 echo
-echo "This file has not been configured yet."
+echo "Updating network channels..."
+echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
+
+random=$RANDOM
+channels=(1 6 11)
+
+let "randomIdx = random % 3"
+let "c = channels[randomIdx]"
+
+echo "Setting new channel to:" $c
 echo
+echo "Updated hostapd.conf:"
+echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
+cd /etc/hostapd
+sudo sed 's/channel=.*/channel='$c'/g' hostapd.conf
+echo 
+echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
+echo "Updated channel successfully! Press Ctrl-C to exist..."
```

Comment:
Tell the students to reboot the pi for these changes to take effect. (theoretically you can "systemctl restart hostapd" but that will kick students off remote desktop. reboot is simpler)